editor/src/components/canvas/canvas-strategies/keyboard-absolute-resize-strategy.tsx

```diff
@@ -104,9 +113,22 @@ export const keyboardAbsoluteResizeStrategy: CanvasStrategy = {
     if (interactionState.interactionData.type === 'KEYBOARD') {
       const accumulatedPresses = accumulatePresses(interactionState.interactionData.keyStates)
       const dragsWithEdges = pressesToVectorAndEdges(accumulatedPresses)
+
+      // Start with the frame as it is at the start of the interaction.
+      let draggedFrame =
```

Comment:
I really dislike that we are using `drag` everywhere in this file for a keyboard strategy

Reply:
I'll see about doing some renaming.

Reply:
I've renamed a bunch of those variables in 12d589a8b.